chainerx_cc/chainerx/python/routines.cc

```diff
@@ -277,6 +278,17 @@ void InitChainerxCreation(pybind11::module& m) {
           "device"_a = nullptr);
 }
 
+void InitChainerxEvaluation(pybind11::module& m) {
+    // evaluation routines
+    m.def("accuracy",
+          [](const ArrayBodyPtr& x1, const ArrayBodyPtr& x2, const absl::optional<int8_t>& ignore_label) {
+              return MoveArrayBody(Accuracy(Array{x1}, Array{x2}, ignore_label));
+          },
+          "x1"_a,
+          "x2"_a,
```

Comment:
Not a strong opinion but how about aligning the argument names to Chainer (`y` and `t`)? Or let me know why we shouldn't.

Reply:
Fixed.